import-service/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "product-service",
+  "version": "1.0.0",
+  "description": "Serverless aws-nodejs-typescript template",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "deploy": "yarn sls deploy",
+    "migrate": "ts-node node_modules/.bin/node-pg-migrate -j ts",
+    "migrate:up": "yarn migrate up",
+    "migrate:down": "yarn migrate down"
+  },
+  "engines": {
+    "node": ">=14.15.0"
+  },
+  "dependencies": {
+    "@babel/preset-typescript": "^7.14.5",
+    "@middy/core": "^1.5.2",
+    "@middy/http-cors": "^2.4.1",
+    "@middy/http-json-body-parser": "^1.5.2",
+    "@types/jest": "^26.0.23",
+    "@types/pg": "^8.6.0",
```

Comment:
It seems that you can move these `@types` to devDependencies

Reply:
The same for `ts-jest` and `aws-sdk-mock`

Reply:
Yes, makes sense, I didn't pay attention on that